unitstatushandler/softwaremanager.go

```diff
@@ -315,6 +323,44 @@ removeLayersLoop:
 	}
 }
 
+func (manager *softwareManager) processRunInstances(desiredInstances []cloudprotocol.InstanceInfo) (needToRun bool) {
```

Comment:
- remove named return value
- I would rename this function to something `needRunInstances` for example.